.github/workflows/build.yml

```diff
@@ -120,6 +120,7 @@ jobs:
           meson setup build \
             -Dwith-init-style=openrc \
             -Dwith-pgp-uam=true \
+            -Dwith-appletalk=true \
```

Comment:
In the future, please list options in alphabetical order.

This might seem like an onerous requirement, however there is a practical reason. When these lists of options get longer, it can be very easy to accidentally list the same option more than once.

This is a real thing that's happened frequently in this codebase, sometimes discovered years later. ;)

No need to take action right now, but please keep it mind the next time you touch this file.